python/spear/urdf/fetch.xml

```diff
@@ -522,20 +522,20 @@
             <inertia ixx="6.98014e-06" ixy="-2.62572e-08" ixz="-1.58206e-08" iyy="4.12665e-05" iyz="-1.57909e-09" izz="3.75763e-05"/>
         </inertial>
         <visual>
-            <origin rpy="0 0 0" xyz="0 -0.101425 0"/>
+            <origin rpy="0 0 0" xyz="0 0.101425 0"/>
             <geometry>
                 <mesh filename="StaticMesh'/UrdfBot/Fetch_V1/Meshes/gripper_finger_link_l.gripper_finger_link_l'"/>
             </geometry>
         </visual>
         <collision>
-            <origin rpy="0 0 0" xyz="0 -0.101425 0"/>
+            <origin rpy="0 0 0" xyz="0 0.101425 0"/>
             <geometry>
                 <mesh filename="StaticMesh'/UrdfBot/Fetch_V1/Meshes/gripper_finger_link_l.gripper_finger_link_l'"/>
```

Comment:
Since we're changing the URDF file anyway, we should also change it to use our custom `spear` tags where appropriate, and make the corresponding changes to the `UrdfBot` plugin at the same time. Since we will be maintaining URDF files and MJCF files going forward, let's name this file `fetch.urdf` instead of `fetch.xml` to avoid confusion.

Reply:
For links, it `convert_urdf.py` already generates a `<spear>` subtag with the `assetpath` attribute pointing to the .uasset file. I think we can keep the URDF joints untouched, because we have to add actuators to the MJCF separately anyway.
Agree with the renaming.

Reply:
Let's check in the updated `fetch.xml` in this PR with the following changes:
- rename `fetch.xml` to `fetch.urdf` to avoid confusion with MJCF that we will generate later
- rename `assetpath` to `unreal_asset`
- fix the indentation (when I ran the code above, the `spear` element was not indented correctly)